### PR TITLE
build: switch to pnpm

### PR DESCRIPTION
**Pull request by enell**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,11 +63,7 @@
     "webpack": "5.87.0",
     "webpack-dev-server": "3.11.3"
   },
-  "workspaces": [
-    "charts/*",
-    "chart-modules/*"
-  ],
   "resolutions": {
-    "**/caniuse-lite": "1.0.30001504"
+    "caniuse-lite": "1.0.30001504"
```
Comment (cbt1) [inline package.json:67]: Don't think `resolutions` works for pnpm. You want to use `{ "pnpm": { "overrides": ... } }`

File: package.json [hunk 2]
```diff
@@ -35,6 +35,7 @@
     "babel-jest": "29.5.0",
     "babel-loader": "9.1.2",
     "chai": "4.3.7",
+    "cross-env": "^7.0.3",
```
Comment (cbt1) [inline package.json:38]: Pin version

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (cbt1) [inline .vscode/settings.json:1]: This file should be added?

File: .circleci/config.yml [hunk 4]
```diff
@@ -89,14 +105,16 @@ jobs:
...
           name: Install Playwright
           command: |
             npx playwright install
       - run:
           name: Install Fonts
           command: ./.circleci/install-fonts.sh
-      - run: yarn run test:rendering
+      - run: pnpm run test:rendering
       - store_test_results:
           path: ./test/rendering/reports/xml/report.xml
```
Comment (cbt1) [inline .circleci/config.yml:119]: This step seems to fail silently on ci. Might be the case on `main/master` as well.
> Unable to save test results from /home/circleci/dist-flow-charts/test/rendering/reports/xml/report.xml
Error error accessing path "/home/circleci/dist-flow-charts/test/rendering/reports/xml/report.xml": stat /home/circleci/dist-flow-charts/test/rendering/reports/xml/report.xml: no such file or directory
Found no test results, skipping

File: package.json (hunk 1 above)
Comment (enell) [reply package.json:67]: It does. I left it like this to keep the change to a minumum but might be better to follow what pnpm suggets

File: package.json (hunk 2 above)
Comment (enell) [reply package.json:38]: done

File: .vscode/settings.json (hunk 3 above)
Comment (enell) [reply .vscode/settings.json:1]: Yes. I don't see a reson not to share workspace settings

**Review by cbt1 — APPROVED** [review]
LGTM

**Pull request merged**